source/Octopus.Tentacle/Services/Capabilities/CapabilitiesServiceV2.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using Octopus.Tentacle.Contracts.Capabilities;
+
+namespace Octopus.Tentacle.Services.Capabilities
+{
+    [Service]
+    public class CapabilitiesServiceV2 : ICapabilitiesServiceV2
+    {
+        public CapabilitiesResponseV2 GetCapabilities()
+        {
+            return new CapabilitiesResponseV2(new List<string>() {"ScriptService", "FileTransferService"});
```

Comment:
Were we not going to return the interfaces e.g. nameof(IScriptService).

This allows a caller to also use the contract names for discovery and reduces a few magic strings.

Reply:
Oh I see that makes more sense.